cpu/k20/Makefile.include

```diff
@@ -0,0 +1,25 @@
+# define the CPU architecture for the k60
+export CPU_ARCH = cortex-m4
+
+# tell the build system that the CPU depends on the Kinetis common files
+export USEMODULE += kinetis_common
+
+# define path to kinetis module, which is needed for this CPU
+export KINETIS_COMMON = $(RIOTCPU)/kinetis_common/
+# CPU depends on the kinetis module, so include it
+include $(KINETIS_COMMON)Makefile.include
+
+#export the CPU model and architecture
+MODEL = $(shell echo $(CPU_MODEL)|tr 'a-z' 'A-Z')
+export CFLAGS += -DCPU_MODEL_$(MODEL)
+ARCH = $(shell echo $(CPU_ARCH) | tr 'a-z-' 'A-Z_')
+export CFLAGS += -DCPU_ARCH_$(ARCH)
```

Comment:
these defiles are being also exported by the `Makefile.include.cortexm_common`, so I think they could be dropped here, right?